packages/snaps-cli/src/cmds/watch/watchHandler.ts

```diff
@@ -40,7 +41,7 @@ export async function watch(argv: YargsArgs): Promise<void> {
     }
 
     try {
-      await bundle(src, outfilePath, argv);
+      await bundle(src, outfilePath, argv, loadConfig().bundler);
```

Comment:
Should we define this as `const config = loadConfig()` outside just in case we need it in the future? 🤔 This goes for multiple files in the PR. Not sure what I think about it.